diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/WhiteSpaceRuleWarnTest.kt

```diff
@@ -615,4 +616,44 @@ class WhiteSpaceRuleWarnTest : LintTestBase(::WhiteSpaceRule) {
             LintError(10, 18, ruleId, tokenWarn("++", 1, null, 0, null), true)
         )
     }
+
+    @Test
+    @Tag(WarningNames.WRONG_WHITESPACE)
+    fun `check whitespaces around braces in lambda example - good`() {
+        lintMethod(
+            """
+                |fun foo() {
+                |    list.map { it.text }
+                |}
+            """.trimMargin()
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.WRONG_WHITESPACE)
+    fun `check whitespaces around braces in lambda example - bad`() {
+        lintMethod(
+            """
+                |fun foo() {
+                |    list.map {it.text}
+                |}
+            """.trimMargin(),
+            LintError(2, 14, ruleId, "${WRONG_WHITESPACE.warnText()} there should be a whitespace after {", true),
+            LintError(2, 22, ruleId, "${WRONG_WHITESPACE.warnText()} there should be a whitespace before }", true)
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.WRONG_WHITESPACE)
+    fun `should not trigger on braces with empty body`() {
+        lintMethod(
+            """
+                |val project = KotlinCoreEnvironment.createForProduction(
+                |   Disposable { },
```

Comment:
Please add also a test when there is no space: `{}`.

Reply:
Added